src/sage/rings/function_field/drinfeld_modules/drinfeld_module.py

```diff
@@ -1061,17 +1272,117 @@ def is_finite(self):
         from sage.rings.function_field.drinfeld_modules.finite_drinfeld_module import FiniteDrinfeldModule
         return isinstance(self, FiniteDrinfeldModule)
 
-    def j_invariant(self):
+    def is_isomorphic(self, psi):
         r"""
-        Return the j-invariant of the Drinfeld module if the rank is
-        two; raise a NotImplementedError otherwise.
+        Return ``True``  whether ``self`` is isomorphic to the Drinfeld module
+        `\psi`.
 
-        Assume the rank is two. Write the generator `\phi_T = \omega +
-        g\tau + \Delta\tau^2`. The j-invariant is defined by
-        `\frac{g^{q+1}}{\Delta}`, `q` being the order of the base field
-        of the function ring. In our case, this field is always finite.
+        EXAMPLES::
 
-        OUTPUT: an element in the base codomain
+            sage: A = GF(5)['T']
+            sage: K.<T> = Frac(A)
+            sage: c = T^4 + 3*T^2 + T
+            sage: phi = DrinfeldModule(A, [T, T^3, T^9, T])
+            sage: psi = DrinfeldModule(A, [T, c^4*T^3, c^(24)*T^9, c^(124)*T])
+            sage: phi.is_isomorphic(psi)
+            True
+            sage: phi0 = DrinfeldModule(A, [T, 1, 1, 1])
+            sage: phi.is_isomorphic(phi0)
```

Comment:
In other parts of the code, we rather used the following naming convention (see `DrinfeldModule` docstring):

```suggestion
            sage: rho = DrinfeldModule(A, [T, 1, 1, 1])
            sage: phi.is_isomorphic(rho)
```